querybean-generator/src/main/java/io/ebean/querybean/generator/PomPluginWriter.java

```diff
@@ -0,0 +1,59 @@
+package io.ebean.querybean.generator;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
+final class PomPluginWriter {
```

Comment:
This isn't a minimalist change ... it's an "Extra Feature" ... these changes make the PR bigger and increase the chance of it not getting approved.  Try and stay minimalist with a view to iterating rather than doing "all things" ... it takes away attention from the issues that stop this PR from being approved.

Reply:
removed